### PR TITLE
Move followings to internal API

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Refactor

## Description

This moves the `followings` endpoints from the external to the internal API as they weren't meant for public consumption.

## Related Tickets & Documents

#6255 

## Added tests?

- [X] yes

## Added to documentation?

- [X] no documentation needed

### CODE REVIEW

File: app/controllers/moderations_controller.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ def article
     @adjustments = TagAdjustment.where(article_id: @moderatable.id)
     @already_adjusted_tags = @adjustments.map(&:tag_name).join(", ")
     @allowed_to_adjust = @moderatable.class.name == "Article" && (current_user.has_role?(:super_admin) || @tag_moderator_tags.any?)
+    @hidden_comments = @moderatable.comments.where(hidden_by_commentable_user: true)
```
Comment (rhymes) [inline app/controllers/moderations_controller.rb]: I think this change belong to the other PR

File: app/views/moderations/mod.html.erb [hunk 2]
```diff
@@ -54,6 +54,20 @@
       This is the equivalent of vomiting on <b>all</b> of this user's articles.
     </p>
   </div>
+  <% if @hidden_comments.present? %>
```
Comment (rhymes) [inline app/views/moderations/mod.html.erb]: same here

File: spec/requests/followings_spec.rb [hunk 3]
```diff
@@ -40,7 +40,7 @@
     end
   end
 
-  describe "GET /api/followings/tags" do
+  describe "GET /afollowings/tags" do
```
Comment (rhymes) [inline spec/requests/followings_spec.rb]: small typo

File: spec/system/articles/moderator_moderates_an_article.rb [hunk 4]
```diff
@@ -24,4 +24,12 @@
     expect(page).to have_selector('button[data-category="vomit"][data-reactable-type="User"]')
     expect(page).to have_selector("button.level-rating-button")
   end
+
+  it "shows hidden comments on /mod" do
```
Comment (rhymes) [inline spec/system/articles/moderator_moderates_an_article.rb]: from the other PR

**Comment by citizen428** [comment]
Thanks @rhymes, I had indeed branched off from the wrong branch. Fixed now, and also fixed the typo.

**Pull request merged**